src/frontends/paddle/include/openvino/frontend/paddle/decoder.hpp

```diff
@@ -0,0 +1,56 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+#include "ngraph/compatibility.hpp"
```

Comment:
why do  we need `ngraph/compatibility.hpp`?

Reply:
it was required for some enable_if templates, but I can't find this code now, so this include has been removed